### ISSUE TITLE
As a contributor, I'd like to read the security policy

### ISSUE THREAD

**Issue by nutjob4life** [NASA-PDS/.github#9]
The new community health files look great! Got a pleasant surprise from them when making a fresh pull-request this morning.

![Screen Shot 2021-09-03 at 9.29.03 AM.png](https://images.zenhubusercontent.com/5d52e426b9ebc816204aa77d/f518ae9a-086d-4d0b-823a-80ef7c3b38a5)

However, once I followed the links, I hit a 404: on the `CONTRIBUTiNG.md`, there's a clickable link to the "security policy" that goes nowhere:

![Screen Shot 2021-09-03 at 9.33.49 AM.png](https://images.zenhubusercontent.com/5d52e426b9ebc816204aa77d/278a4a6b-e6e9-46c7-8fd1-023ecb6d2f39)

You do get a cute Octocat though! Since the `SECURITY.md` is specific for each repository (because it typically lists supported versions), we could

- Reword this to mention checking the `SECURITY.md` file in a relevant repository without linking to it.
- Rewrite this to mention emailing a `pds-security@jpl.nasa.gov` email list; a lot of other organizations prefer to have security problems sent by encrypted email rather than publicly tracked for the obvious implications (with the obvious need for a public key to also be available for `pds-security@jpl.nasa.gov`).
- Drop this bullet.

Not sure what the right approach is.

May the force be with you 🐙🐱

**Comment by MJJoyce** [comment]
@jordanpadams, you have a preference here? I'm throwing up a PR requesting an email to go to the `pds-pmc@` but I'm happy to update with something else.

**Comment by jordanpadams** [comment]
> @jordanpadams, you have a preference here? I'm throwing up a PR requesting an email to go to the `pds-pmc@` but I'm happy to update with something else.

@MJJoyce this is fine by me

**Comment by nutjob4life** [comment]
> throwing up a PR

PHRASING 🤢
